frontend/packages/kubevirt-plugin/src/components/vm-status/vm-status.tsx

```diff
@@ -102,6 +115,11 @@ export const VMStatus: React.FC<VMStatusProps> = ({
   )}`; // to default tab
   const additionalText = verbose ? getAdditionalImportText(statusDetail.pod) : null;
 
+  const unpauseVMI = async (event) => {
+    event.preventDefault();
+    await unpauseVM(vmi);
```

Comment:
shouldn't we do something here when this fails?

Reply:
the error should be handled in some way. Lets do it as a follow up @pcbailey